packages/mobile/App.svelte

```diff
@@ -44,7 +44,14 @@
         document.dir = $localeDirection
     }
 
-    $: $isLocaleLoaded, nativeSplash.hide()
+    $: if ($isLocaleLoaded) {
+        hideSplashScreen()
+    }
```

Comment:
I guess, you have to add a at least a catch statement to handle the warning?

Reply:
🤔 I can't see a warning, where is it?